### ISSUE TITLE
Bundle PSP package onto the ingress node

### ISSUE THREAD

**Issue by jpoon** [jpoon/aurora_deploy#9]
We haven't bundled the PSP package onto the ingress node & therefore I can't test an e2e dataflow. This is most impactful as I can't verify that Pachyderm is correctly configured to use Azure Storage. @moadben any chance of bundling this?

**Comment by brendanacassidy** [comment]
I think @moadben is working on this as we speak

**Comment by moadben** [comment]
Gonna update the psp code to use SAS so that we can just deploy the commands with no fear of azure acc name/key environment variables. However since I can't use the go sdk for this I'm gonna have to update my blob package to send get requests and parse the xml(I believe it returns xml) that's returned. Might take a while.

Tl;dr: updating psp to use sas, need to update code to read from API requests.

**Comment by moadben** [comment]
We've tested and submitted our changes, the sas key authentication has been vendored into our parse repo and is fully functional.
